ens-improvement-proposals/ensip-15/tests.json

```diff
@@ -11,8 +11,17 @@
 	},
 	{
 		"name": "",
+		"comment": "Empty"
```

Comment:
The exact empty string should be no error.  It should be interpreted as 0-labels instead of 1 empty label.

https://github.com/adraffy/ensip-15/blob/master/ens-improvement-proposals/ensip-15-normalization-standard.md?plain=1#L242